native/protocol/src/x3dh.rs

```diff
@@ -207,7 +207,7 @@ pub fn initiate_recv(
     sender_identity_key: &VerifyingKey,
     ephemeral_key: X25519PublicKey,
     otk: Option<X25519StaticSecret>,
-    ciphertext: &str,
+    ciphertext: Vec<u8>,
```

Comment:
This moves ownership of the ciphertext from the caller into this function. I think this should probably be ```&[u8]``` instead of ```Vec<u8>```